src/classes/UTIL_OrgTelemetry_SObject_BATCH.cls

```diff
@@ -65,6 +65,14 @@ public class UTIL_OrgTelemetry_SObject_BATCH implements Database.Batchable<SObje
         OPPORTUNITIES
     }
 
+    /** @description Limit for query locator. Property is public to allow for testing */
+    public Integer chunkSize = 10000000;
```

Comment:
Should this be `@TestVisible private` if it's just needed for tests?

Reply:
I meant testing from the developer console for QA, I'll update the comment